test/fixtures/sanitation/response-body-attribute.js

```diff
@@ -21,12 +21,18 @@ hooks.after('Resource > Update Resource', (transaction, done) => {
   const validationOutput = transaction.test.results.body;
 
   const errors = [];
-  for (let i = 0; i < validationOutput.results.length; i++) {
-    if (validationOutput.results[i].pointer !== '/token') {
-      errors.push(validationOutput.results[i]);
+  // TODO
+  // Why do we even assert Gavel's output here?
+  // If custom transformations are needed:
+  // 1. Create a transformer function.
+  // 2. Test is on _assumed_ Gavel output.
+  // 3. Assert transformed output in sanitization tests, if any.
+  for (let i = 0; i < validationOutput.errors.length; i++) {
+    if (validationOutput.errors[i].pointer !== '/token') {
+      errors.push(validationOutput.errors[i]);
```

Comment:
This code is a (tested) example linked from docs, which shows how to sanitize a whole attribute in message body. First it is removed from the body, then from the schema, then from the gavel result. It mutates the data to remove the attribute, so then when Dredd reports this data (e.g. to Apiary), it does not contain any information about the attribute. Since Gavel result contains pointers to attributes and the whole validation result is then sent over to reporters, and potentially to Apiary, it is also sanitized this way.